guides/common/modules/con_upgrading-prerequisites.adoc

```diff
@@ -16,6 +16,11 @@ Upgrading {SmartProxy} takes approximately 10{range}30 minutes.
 
 * Ensure that you have sufficient storage space on your server.
 For more information, see {InstallingServerDocURL}Preparing_your_Environment_for_Installation_{project-context}[Preparing your Environment for Installation] in _{InstallingServerDocTitle}_ and {InstallingSmartProxyDocURL}preparing-environment-for-capsule-installation[Preparing your Environment for Installation] in _Installing {SmartProxyServer}_.
+ifndef::foreman-deb[]
+* Ensure that you have at least the same amount of free space on `{postgresql-lib-dir}` as that consumed by `{postgresql-data-dir}`.
+Upgrading to {Project} {ProjectVersion} involves a PostgreSQL 12 to PostgreSQL 13 upgrade.
+The contents of `{postgresql-lib-dir}` are backed up during the PostgreSQL upgrade. 
```

Comment:
I think technically it backs up the data dir, not the lib dir. That's which is why you need the same amount of space as the data dir.
```suggestion
The contents of `{postgresql-data-dir}` are backed up during the PostgreSQL upgrade. 
```